super_gui/src/components/DoorManUI.tsx

```diff
@@ -31,14 +51,39 @@ const DoorManUI = () => {
     // Mock implementation
   };
 
-  // Mock function to discard package
-  const discardPackage = (id: number) => {
-    // Mock implementation
+  // PATCH Packages Status
+  const discardPackage = async ({ id }: { id: number }) => {
+    try {
+      const res = await fetch(`http://localhost:8080/locker/${id}`, {
+        method: "PATCH",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          apartmentNumber: null,
+        }),
+      });
+
+      if (res.ok) {
+        const data = await res.json();
+        setLockers((prevLockers) =>
```

Comment:
with one users, this is oke. If you have multiple users which can work at the same time, you can better do a new get request for all lockers instead of changing the locker in the current array of lockers. It could be that someone else also changed something at the same time, which means your working with old data.